include/cvc5/cvc5_skolem_id.h

```diff
@@ -171,6 +171,40 @@ enum ENUM(SkolemId)
    * - Sort: ``Real``
    */
   EVALUE(TRANSCENDENTAL_SINE_PHASE_SHIFT),
+  /**
+   * Used to reason about virtual term substitution. This term represents
+   * an infinitessemal.
+   *
+   * - Number of skolem indices: ``0``
+   * - Sort: ``Real``
+   */
+  EVALUE(ARITH_VTS_DELTA),
+  /**
+   * Used to reason about virtual term substitution. This term represents
+   * an infinitessemal.
+   *
+   * - Number of skolem indices: ``0``
+   * - Sort: ``Real``
+   */
+  EVALUE(ARITH_VTS_DELTA_FREE),
+  /**
+   * Used to reason about virtual term substitution. This term represents
+   * infinity.
+   *
+   * - Number of skolem indices: ``0``
+   *   - ``1:`` A term that represents an arithmetic sort (Int or Real).
+   * - Sort: The sort given by the index.
+   */
+  EVALUE(ARITH_VTS_INFINITY),
+  /**
+   * Used to reason about virtual term substitution. This term represents
+   * infinity.
+   *
+   * - Number of skolem indices: ``0``
+   *   - ``1:`` A term that represents an arithmetic sort (Int or Real).
+   * - Sort: The sort given by the index.
+   */
+  EVALUE(ARITH_VTS_INFINITY_FREE),
```

Comment:
ditto. How is this different from `ARITH_VTS_INFINITY`?

Reply:
I will doc.